app/javascript/utils/Patient.js

```diff
@@ -59,4 +64,28 @@ function formatRace(patient) {
   return raceArray.length === 0 ? '--' : raceArray.join(', ');
 }
 
-export { formatName, formatNameAlt, formatPhoneNumber, formatRace };
+function formatDateOfBirth(data) {
+  const rowData = data.rowData;
+  if (!!rowData?.dob && moment(rowData.dob, 'YYYY-MM-DD').isAfter(moment().subtract(18, 'years'))) {
```

Comment:
I'd strongly prefer to have the DOB passed into this function, then you it would be a lot simpler to follow

Reply:
Looking into it, there's two issues with this: first, because of how CustomTable is written and takes in functions, it's difficult to have just this function take in a date: it's being passed as an argument to a function which calls it on the entire data. We use the full data as an argument in related functions in utils/DateTime.js with formatTimestamp and formatDate. I agree that this is a weird way of doing it, but I think that replacing it would be a more extensive job and in the meantime we should consider just renaming the function for now to more accurately reflect what it does.

Reply:
just have a function in both the move to household and patient table components that is formatDOB or something and then call this from there with the date argument. would be much cleaner

Reply:
Whoops I missed this when I did my review, but I would also like to see this change. Here's my comment: https://github.com/SaraAlert/SaraAlert/pull/915#discussion_r653827574